guides/common/modules/proc_installing-the-discovery-service.adoc

```diff
@@ -9,7 +9,9 @@ Complete the following procedure to enable the Discovery service on {SmartProxyS
 ifdef::satellite,orcharhino[]
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {package-install-project} foreman-discovery-image tfm-rubygem-smart_proxy_discovery
+# {foreman-installer} \
+--enable-foreman-proxy-plugin-discovery \
+--foreman-proxy-plugin-discovery-install-images=false
 ----
 endif::[]
 ifdef::foreman-el,katello[]
```

Comment:
While you're at it, mind changing this too?
```suggestion
ifndef::satellite,orcharhino[]
```

Reply:
I never mind at all, Ewoud :heart: 

Addressed :+1: